Documentation/cli/starlark.md

```diff
@@ -321,3 +321,19 @@ def command_flaky(args):
 		restart(Rerecord=True)
 
 ```
+
+## Passing a struct as an argument
+
+Struct literals can be passed to built-ins as JSON. For example, the following snippet passes
+in an [api.EvalScope](https://pkg.go.dev/github.com/go-delve/delve@v1.20.2/service/api#EvalScope)
```

Comment:
Do we need to write the specific version `v1.20.2` here?
```suggestion
in an [api.EvalScope](https://pkg.go.dev/github.com/go-delve/delve/service/api#EvalScope)
```

The same is below.

Reply:
I think making links to fixed versions is generally a good idea (so that the link still works if the source code changes but the example wasn't updated).
However, other links in this document do not have them, so I've removed them for consistency.